api/src/controllers/team.js

```diff
@@ -310,6 +315,102 @@ async function updateTeam(req, res, next) {
   }
 }
 
+/**
+  * Clear the team Data from the database on /team/:team_id
+  * @param {} req request object, containing team id in the url
+  * @param {*} res response object, the deleted team document
+  * @returns 200: teamn is deleted
+  * @returns 404: team is not found
+  * @returns 400: team id is not in a valid hexadecimal format
+  */
+async function clearTeam(req, res, next) {
+  try {
+    const { team_id: _id } = req.params;
+    const { isDeleteFlag } = req.body;
+    await HomePage.deleteMany({ teamId: _id });
+    await Website.deleteMany({ teamId: _id });
```

Comment:
Just an FYI that each team should only have one homepage and website doc so you don't actually need to use deleteMany() here but that's not an issue. :)